migrations/0001_create_table.sql

```diff
@@ -0,0 +1,153 @@
+    "organization" TEXT NOT NULL,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    CONSTRAINT "MemberActiveExternal_memberId_fkey" FOREIGN KEY ("memberId") REFERENCES "Member" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+-- CreateTable
+CREATE TABLE "MemberAlumni" (
+    "memberId" TEXT NOT NULL PRIMARY KEY,
+    "graduatedYear" INTEGER NOT NULL,
+    "oldRole" TEXT NOT NULL,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    CONSTRAINT "MemberAlumni_memberId_fkey" FOREIGN KEY ("memberId") REFERENCES "Member" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+-- CreateTable
+CREATE TABLE "Payment" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "payerId" TEXT NOT NULL,
+    "receiverId" TEXT NOT NULL,
+    "hasApproved" BOOLEAN NOT NULL DEFAULT false,
+    "approverId" TEXT,
+    "receivedAt" DATETIME NOT NULL,
+    "approvedAt" DATETIME,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT "Payment_payerId_fkey" FOREIGN KEY ("payerId") REFERENCES "Member" ("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    CONSTRAINT "Payment_receiverId_fkey" FOREIGN KEY ("receiverId") REFERENCES "Member" ("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    CONSTRAINT "Payment_approverId_fkey" FOREIGN KEY ("approverId") REFERENCES "Member" ("id") ON DELETE SET NULL ON UPDATE CASCADE
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Member_id_key" ON "Member"("id");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Member_subject_key" ON "Member"("subject");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Member_email_key" ON "Member"("email");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberBase_memberId_key" ON "MemberBase"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberStatus_memberId_key" ON "MemberStatus"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberSensitive_memberId_key" ON "MemberSensitive"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberActive_memberId_key" ON "MemberActive"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberActiveInternal_memberId_key" ON "MemberActiveInternal"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberActiveInternal_studentId_key" ON "MemberActiveInternal"("studentId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberActiveExternal_memberId_key" ON "MemberActiveExternal"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "MemberAlumni_memberId_key" ON "MemberAlumni"("memberId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Payment_id_key" ON "Payment"("id");
+
+-- Seed
+INSERT INTO "Member" ("id", "subject", "email", "securityRole", "updatedAt") VALUES
+('0188c0f2-8e47-11ec-b909-0242ac120002', 'Mathematics', 'math@example.com', 'admin', CURRENT_TIMESTAMP),
+('0188c0f2-8e47-11ec-b909-0242ac120003', 'Science', 'science@example.com', 'user', CURRENT_TIMESTAMP),
+('0188c0f2-8e47-11ec-b909-0242ac120004', 'History', 'history@example.com', 'user', CURRENT_TIMESTAMP);
```

Comment:
`app/models/member.ts` 的に `admin` `user` は大文字では?
```ts
export const SECURITY_ROLE = [
  'DEFAULT',
  'MEMBER',
  'READ',
  'PAYMENT_APPROVE',
  'OWNER',
] as const;
const zSecurityRoles = z.enum(SECURITY_ROLE);
```

```suggestion
INSERT INTO "Member" ("id", "subject", "email", "securityRole", "updatedAt") VALUES
('0188c0f2-8e47-11ec-b909-0242ac120002', 'Mathematics', 'math@example.com', 'ADMIN', CURRENT_TIMESTAMP),
('0188c0f2-8e47-11ec-b909-0242ac120003', 'Science', 'science@example.com', 'USER', CURRENT_TIMESTAMP),
('0188c0f2-8e47-11ec-b909-0242ac120004', 'History', 'history@example.com', 'USER', CURRENT_TIMESTAMP);
```

`http://localhost:5173/test` で↓って出る
> ZodError: [
>   {
>     "received": "admin",
>     "code": "invalid_enum_value",
>     "options": [
>       "DEFAULT",
>       "MEMBER",
>       "READ",
>       "PAYMENT_APPROVE",
>       "OWNER"
>     ],
>     "path": [],
>     "message": "Invalid enum value. Expected 'DEFAULT' | 'MEMBER' | 'READ' | 'PAYMENT_APPROVE' | 'OWNER', received 'admin'"
>   }
> ]

Reply:
確かに大文字・小文字が間違っていました！
…… でも, `ADMIN` っぽかったけど, 普通に `OWNER` でした……